### ISSUE TITLE
Still under active development?

### ISSUE THREAD

**Issue by claman** [thisisparker/cursewords#46]
Most of the issues and pull requests are 3-5 years old.

**Comment by thisisparker** [comment]
Would you mind clarifying your question?

**Comment by claman** [comment]
Yes, sorry for not being clear (and for phrasing my question rudely).

The program still works very well at this point, about 3 years after its latest release -- I'm not asking out of a concern for unaddressed bugs, etc. And at the same time, it's been about 3 years since the latest release. So I guess the better question would be: do you have any plans for an update release sometime soon? Thinking especially of some of the QOL suggestions, etc.

Again, my apologies for phrasing my initial question poorly.
